packages/manager/modules/pci/src/projects/project/project.constants.js

```diff
@@ -255,6 +255,8 @@ export const DISCOVERY_PROJECT_ACTIVATION_PAYLOAD = {
   quantity: 1,
 };
 
+export const DISCOVERY_PROJECT_ACTIVATION_PLANCODE = 'project.2018';
```

Comment:
I would not call this as a `DISCOVERY_PROJECT_ACTIVATION_PLANCODE` rather this can be `PCI_FULL_PROJECT_PLANCODE`.
Also, it is hardcoded in `isManuallyReviewedByAntiFraud` function on the same file.Can you change that as well?